aws/resource_aws_organizations_organization_test.go

```diff
@@ -354,3 +354,91 @@ func testFlattenOrganizationsRootPolicyTypes(t *testing.T, index int, result []m
 		}
 	}
 }
+
+func testAccAwsOrganizationsOrganizationRecreated(t *testing.T, before, after *organizations.Organization) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		if *before.Id == *after.Id {
```

Comment:
Nit: we should unwrap these with the AWS Go SDK's method `aws.StringValue()` to prevent any unwanted panics